scripts/irods/test/test_iadmin.py

```diff
@@ -486,6 +486,20 @@ def test_iadmin_mkuser(self):
         self.admin.assert_icommand("iadmin mkuser lando invalid_user_type",
                                    'STDERR_SINGLELINE', "CAT_INVALID_USER_TYPE")  # should be rejected
 
+    @unittest.skipIf(IrodsConfig().version_tuple < (4, 3, 2), "Only applies to 4.3.2 and above.")
+    def test_new_clients_cannot_rmuser_groups_nor_rmgroup_users__issue_7380(self):
+        self.admin.assert_icommand("iadmin mkuser testuser rodsuser")
+        self.admin.assert_icommand("iadmin mkgroup testgroup")
+
+        try:
+            self.admin.assert_icommand("iadmin rmuser testgroup",
+                                       'STDERR_SINGLELINE', "CAT_INVALID_USER")
+            self.admin.assert_icommand("iadmin rmgroup testuser",
+                                       'STDERR_SINGLELINE', "CAT_INVALID_GROUP")
```

Comment:
Consider including more characters in the expected output string. As it is, if something like `OTHER_CAT_INVALID_USER_STUFF` was returned, the assertion for L495 would pass.

This helps make the test more resilient to future changes.